docs/getting-started/security-ui.asciidoc

```diff
@@ -110,3 +110,50 @@ The Administration page allows you to view and manage hosts that are running End
 
 [role="screenshot"]
 image::images/admin-ui.png[]
+
+[discrete]
+[[timeline-accessibility-features]]
+== Accessibility features
+
+Accessibility features, such as keyboard focus and screen reader support, are built into the Elastic Security UI. These features offer additional ways for you to navigate the UI and interact with the application.
+
+[discrete]
+[[draggable-timeline-elements]]
+=== Interact with draggable elements
+
+Use your keyboard to interact with draggable elements in the Elastic Security UI:
+
+* Select a draggable element to give it keyboard focus.
+
+[role="screenshot"]
+image::images/timeline-ui-accessiblity-keyboard-focus.gif[width=100%][height=100%][Demo that shows how to give a draggable element keyboard focus]
+
+* Press *Enter* on an element with keyboard focus to display its menu.
+
+[role="screenshot"]
+image::images/timeline-ui-accessiblity-display-menu.gif[width=100%][height=100%][Demo that shows how to display an element's menu]
+
+* Press the spacebar on your keyboard to begin dragging a field to a different location and the directional keys to move it to a drop area. Press the spacebar again to drop the field.
+
+[role="screenshot"]
+image::images/timeline-ui-accessiblity-drag-and-drop.gif[width=100%][height=100%][Demo that shows how to drag and drop an element to another location in the Elastic Security UI]
+
+[discrete]
+[[timeline-tab]]
+=== Navigate the Elastic Security UI
+Use your keyboard to navigate through rows, columns, and menu options of elements in the Elastic Security UI:
+
+* Use the directional arrows to move keyboard focus right, left, up, and down.
+
+[role="screenshot"]
+image::images/timeline-ui-accessiblity-directional-arrows.gif[width=100%][height=100%][Demo that shows how to drag and drop an element to another location in the Elastic Security UI]
+
```

Comment:
Consider adding the following instruction and animated gif of the Alerts table here, (just below the previous instructions and gif that described navigation of cells via the arrow keys):

- When a table cell, (selected via the arrow keys or mouse), contains multiple items, such as buttons or field names, press the `Tab` key to focus each item in the cell, as shown in the following gif:

![actions-menu](https://user-images.githubusercontent.com/4459398/111349705-fddec580-8646-11eb-8597-22fdc7e47e00.gif)